bridge/client/app/_components/ktb-sequence-tasks-list/ktb-sequence-tasks-list.component.ts

```diff
@@ -1,24 +1,19 @@
-import { Component, HostBinding, Input, OnDestroy, OnInit, ViewEncapsulation } from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, HostBinding, Input } from '@angular/core';
+import { Router } from '@angular/router';
 import { Location } from '@angular/common';
 import { Trace } from '../../_models/trace';
 import { DateUtil } from '../../_utils/date.utils';
-import { takeUntil } from 'rxjs/operators';
-import { Subject } from 'rxjs';
 
 @Component({
-  selector: 'ktb-sequence-tasks-list[tasks][stage]',
+  selector: 'ktb-sequence-tasks-list[tasks][focusedEventId]',
   templateUrl: './ktb-sequence-tasks-list.component.html',
   styleUrls: ['./ktb-sequence-tasks-list.component.scss'],
-  encapsulation: ViewEncapsulation.None,
-  preserveWhitespaces: false,
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class KtbSequenceTasksListComponent implements OnInit, OnDestroy {
+export class KtbSequenceTasksListComponent {
   @HostBinding('class') cls = 'ktb-sequence-tasks-list';
-  public _tasks: Trace[] = [];
-  public _stage?: string;
-  public _focusedEventId?: string;
-  private readonly unsubscribe$ = new Subject<void>();
+  private _tasks: Trace[] = [];
+  private _focusedEventId?: string;
   private currentScrollElement?: HTMLDivElement;
 
   @Input()
```

Comment:
You might need to move the `@Input()` to the setter for the change detection `OnPush` to work correctly.